docs-chef-io/content/workstation/cookstyle/_index.md

```diff
@@ -247,3 +247,19 @@ standard. Include this file in the .rubocop.yml file by adding
 `inherit_from: .rubocop_todo.yml` to the top of the .rubocop.yml file.
 
 {{< /note >}}
+
+## Cookstyle Comments in Code
+
+You can also use `cookstyle` specific comments in your cookbook code to enable or disable cops instead of the standard `rubocop` comments. We think that it will be easier to understand the cops that you intend to control if you use `cookstyle` comments. You can continue to use the existing `rubocop` comments, if you prefer them, since both types of comments will be honored by Cookstyle.
```

Comment:
```suggestion
Use Cookstyle-specific comments in your cookbook code to enable or disable cops instead of the standard RuboCop comments.
If you prefer them, you can also use RuboCop comments to enable and disable cops.
```